docs/csharp/getting-started/index.md

```diff
@@ -1,51 +1,47 @@
+- For detailed information about specific aspects of the C# language, see the [C# Reference](../language-reference/index.md).
+- For more information about LINQ, see [LINQ (Language-Integrated Query)](../programming-guide/concepts/linq/index.md).
 
-   Visual Studio Code is a free code editor optimized for building and debugging modern web and cloud applications. It supports IntelliSense and is available for Linux, macOS, and Windows.
+## .NET Platform Architecture
 
-   This topic shows you how to create and run a simple Hello World application with Visual Studio Code and .NET Core.
+C# programs run on .NET, a virtual execution system called the common language runtime (CLR) and a unified set of class libraries. The CLR is the commercial implementation by Microsoft of the common language infrastructure (CLI), an international standard. The CLIis the basis for creating execution and development environments in which languages and libraries work together seamlessly.
 
-## Related sections
+Source code written in C# is compiled into an [intermediate language (IL)](../../standard/managed-code.md) that conforms to the CLI specification. The IL code and resources, such as bitmaps and strings, are stored in an assembly, typically with an extension of .dll. An assembly contains a manifest that provides information about the assembly's types, version, and culture.
 
-- [C# Programming Guide](../programming-guide/index.md)
+When the C# program is executed, the assembly is loaded into the CLR. The CLR performs Just-In-Time (JIT) compilation to convert the IL code to native machine instructions. The CLR provides other services related to automatic garbage collection, exception handling, and resource management. Code that's executed by the CLR is sometimes referred to as "managed code", in contrast to "unmanaged code", which is compiled into native machine language that targets a specific system.
 
-    Provides information about C# programming concepts, and describes how to perform various tasks in C#.
+Language interoperability is a key feature of .NET. IL code produced by the C# compiler conforms to the Common Type Specification (CTS). IL code generated from C# can interact with code that was generated from the .NET versions of F#, Visual Basic, C++, or any of more than 20 other CTS-compliant languages. A single assembly may contain multiple modules written in different .NET languages, and the types can reference each other as if they were written in the same language.
 
-- [C# Reference](../language-reference/index.md)
+In addition to the run time services, .NET also includes an extensive library of over 4000 classes. The .NET library is organized into namespaces that provide a wide variety of useful functionality for everything from file input and output to string manipulation to XML parsing, to web application frameworks to Windows Forms controls. The typical C# application uses the .NET class library extensively to handle common "plumbing" chores.
```

Comment:
```suggestion
In addition to the run time services, .NET also includes an extensive library of over 4,000 classes. The .NET library is organized into namespaces that provide a wide variety of useful functionality for everything from file input and output to string manipulation to XML parsing, to web application frameworks to Windows Forms controls. The typical C# application uses the .NET class library extensively to handle common "plumbing" chores.
```

Is 4,000 classes accurate - I'd imagine that it's WAY more now? Also, you mention the ".NET library" here as if there is only one, which one are you referring to - I'm confused by that? Do you mean the runtime? If so the, there are multiple implementations...I'm not sure how to update this. Maybe a link to the library?